modules/@ergo/users/components/Grid/Layout/Table/Cells/Data/GridTextHintDataCell.vue

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
Is this file is in the correct place? All components related with cell data are included in @Core module.

Reply:
It's related to @users module only. I'll remember to do sth about it